app/src/main/java/app/src/main/java/CliGameModel.java

```diff
@@ -0,0 +1,537 @@
+package app.src.main.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+import java.io.IOException; // For handling IOException if needed
+
+
+import app.src.main.java.CliGameController;
+
+
+
+public class CliGameModel extends helpers {
+    /**
+     * This is possible points a current puzzle can have.
```

Comment:
I appreciate the comments, but in this case, I don't believe them to be of much value. I believe we did a good job of creating var names. I think they describe what they store. Removing the comments would make this file look less crowded. We can talk about it if you would like, if not -> remove the comment blocks and keep white space to one line OR use this format //... on comments that are one-liners. Thanks!